src/libs/ReportUtils.js

```diff
@@ -661,20 +667,49 @@ function buildOptimisticIOUReport(ownerEmail, recipientEmail, total, chatReportI
     };
 }
 
+function getIOUReportActionMessage(type, total, participants, comment, currency) {
+    const amount = NumberFormatUtils.format(preferredLocale, total / 100, {style: 'currency', currency});
+
+    const displayNames = _.map(participants, participant => lodashGet(participant, 'text', participant.login));
+    const from = displayNames.length < 3
+        ? displayNames.join(' and ')
+        : `${displayNames.slice(0, -1).join(', ')}, ${Localize.translateLocal('common.and')} ${_.last(displayNames)}`;
+
+    let iouMessage;
+    switch (type) {
+        case CONST.IOU.REPORT_ACTION_TYPE.CREATE:
+            iouMessage = Localize.translateLocal('iou.requestedFrom', {amount, from, comment});
```

Comment:
We dont do the translation in the backend do we? so if this is created in the backend it will be in English and we wont translate it to Spanish in the client

Reply:
We don't translate in the server. Now that you mentioned, it makes sense to not translate it here either, otherwise the user would see this changing back to English when they refresh the App.